common/views/components/ChapterIndicator/ChapterIndicator.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+const colorDataUris = {
+  white: 'P///wAAACH5BAAAAAAALAAAAAABAAQAAAIChFEAOw==',
+  red: 'P9DVQAAACH5BAAAAAAALAAAAAABAAQAAAIChFEAOw==',
+  purple: 'IxHmQAAACH5BAAAAAAALAAAAAABAAQAAAIChFEAOw==',
+  turquoise: 'Fy4vwAAACH5BAAAAAAALAAAAAABAAQAAAIChFEAOw==',
+  orange: 'Oh1AAAAACH5BAAAAAAALAAAAAABAAQAAAIChFEAOw=='
+};
+
+type Props = {|
+  position: number,
+  showSingle?: boolean,
+  showTotal?: boolean,
+  color: string,
```

Comment:
You could type this to the keys of the `colorDataUris` to avoid the below exploding if you pass it an invalid colour.